helm_deploy/hmpps-prisoner-profile/values.yaml

```diff
@@ -60,6 +61,9 @@ generic-service:
     sqs-hmpps-audit-secret:
       AUDIT_SQS_QUEUE_URL: "sqs_queue_url"
       AUDIT_SQS_QUEUE_NAME: "sqs_queue_name"
+    os-places-api-secret:
+      OS_PLACES_API_KEY: "API_KEY"
```

Comment:
Did this need to have its own secret or could it be added to the `hmpps-prisoner-profile` secret?

Reply:
I'm just on auto pilot for using the secret manager integration! Are the `hmpps-prisoner-profile` secrets just added manually?

I have no objection to moving this one into the existing secret

Reply:
Done :)